### PR TITLE
Indicate unsync on the app side

### LINKED ISSUE #10: Indicate unsync on the app side
Find `last-move` string and reuse for peripheral and centrall diff

### PR DESCRIPTION
**Pull request by vovagorodok**

### CODE REVIEW

File: src/chessground/render.ts [hunk 1]
```diff
@@ -264,12 +264,35 @@ function addSquare(squares: Map<Key, string>, key: Key, klass: string) {
...
+  if (d.peripheral.isSynchronized || arePeripheralPiecesEmpty) {
+    if (d.lastMove && d.highlight.lastMove) {
+      addSquare(squares, d.lastMove[0], 'last-move')
+      addSquare(squares, d.lastMove[1], 'last-move')
+    }
+    if (d.check && d.highlight.check) {
+      addSquare(squares, d.check, 'check')
+    }
+  } else {
+    for (const [key, crntralPiece] of centralPieces) {
```
Comment (dkwach) [inline src/chessground/render.ts]: typo: crntral ?

Comment (vovagorodok) [reply src/chessground/render.ts]: Fixed

File: src/chessground/fen.ts [hunk 2]
```diff
@@ -27,6 +27,36 @@ const letters = {
   king: 'k'
 }
```
Comment (dkwach) [inline src/chessground/fen.ts:29]: Cen we use read function to avoid almost the same code

File: src/chessground/fen.ts [hunk 3]
```diff
@@ -27,6 +27,36 @@ const letters = {
...
 
+export function readPeripheral(fen: string): cg.PeripheralPieces {
```
Comment (dkwach) [inline src/chessground/fen.ts]: Maybe parsePeripherialFen

File: src/chessground/render.ts [hunk 4]
```diff
@@ -264,12 +264,35 @@ function addSquare(squares: Map<Key, string>, key: Key, klass: string) {
 
 function computeSquareClasses(d: State): Map<Key, string> {
```
Comment (dkwach) [inline src/chessground/render.ts:265]: It is quite long now. What about splitting into smaller functions

```

File: src/externalDevice.ts [hunk 5]
```diff
@@ -20,13 +22,19 @@ export default {
     bluetooth.protocol().onMoveRejectedByCentral()
   },
   sendMoveToCentral(orig: Key, dest: Key, prom?: Role) {
-    if (callback)
-      callback(orig, dest, prom)
+    if (onPeripheralMove)
```
Comment (dkwach) [inline src/externalDevice.ts]: what about null safety call?

File: src/chessground/fen.ts (hunk 3 above)
Comment (vovagorodok) [reply src/chessground/fen.ts]: Hmm, here I'm trying to be consistent with function:
``` 
export function read(fen: string): cg.Pieces {
```

File: src/chessground/render.ts (hunk 4 above)
Comment (vovagorodok) [reply src/chessground/render.ts:265]: Let's split it in the future, maybe after adding more indications

File: src/externalDevice.ts (hunk 5 above)
Comment (vovagorodok) [reply src/externalDevice.ts]: Fixed

File: src/chessground/fen.ts (hunk 2 above)
Comment (vovagorodok) [reply src/chessground/fen.ts:29]: Maybe by using functors and template read functions. Lets leave it for refactor in future

Comment (dkwach) [reply src/chessground/fen.ts:29]: ok

File: src/chessground/fen.ts (hunk 3 above)
Comment (dkwach) [reply src/chessground/fen.ts]: I would rename read as well

Comment (vovagorodok) [reply src/chessground/fen.ts]: Fixed

**Pull request merged**